tests/unit/test_api.py

```diff
@@ -17,7 +17,10 @@
 
 """Test api."""
 
+from invenio_search import current_search_client
```

Comment:
Tests seems small compare to the changes...

Reply:
Generally more test must be added.